guides/installation/requirements.md

```diff
@@ -0,0 +1,182 @@
+# Requirements
+
+Before installing Shopware 6, you should take a quick look at the requirements to check if your local environment is capable of running it.
+
+## Operating System
+
+Shopware 6 is currently only supported on linux and macOS setups.
+Windows is only supported inside wsl.
+
+## Versions
+
+You can use these commands to check your actual environment:
+
+* `php -v`: Show CLI PHP version
+* `php -m`: Show CLI PHP modules
+* `php -i | grep memory_limit`: Show your actual CLI PHP memory limit
+* `composer -v`: Show your actual composer version
+* `node -v`: Show you actual Node version
+* `npm -v`: Show you actual NPM version
+
+**PHP**
+
+* Tested on 7.4.3, 8.0 and 8.1
+* `memory_limit` 512M minimum
+* `max_execution_time` 30 seconds minimum
+* Extensions:
+  * ext-curl
+  * ext-dom  
+  * ext-fileinfo  
+  * ext-gd  
+  * ext-iconv  
+  * ext-intl  
+  * ext-json  
+  * ext-libxml  
+  * ext-mbstring  
+  * ext-openssl  
+  * ext-pcre  
+  * ext-pdo  
+  * ext-pdo\_mysql  
+  * ext-phar  
+  * ext-simplexml
+  * ext-sodium
+  * ext-xml  
+  * ext-zip  
+  * ext-zlib
+* Composer 2.0 or higher
+
+**SQL**
+
+* Tested on MySQL 5.7.21, and 8.0
+  * Only MySQL 8.0.20 in specific, is not compatible
+* Tested on MariaDB 10.3.22, 10.4 and 10.5
+  * MariaDB 10.3.29, 10.4.19, 10.5.10 are not compatible at the moment
+
+**JavaScript**
+
+* Node.js 12.21.0 or higher
+* NPM 6.5.0 or higher
+
+## Webserver
```

Comment:
I would like to make a Recommended Setup section

```
Webserver: Caddy
PHP: 8.1 (6.5  8.2)
Node: 16 (6.5 18)
Search: Opensearch 2.5.0
Queue: Rabbitmq
```

Reply:
Maybe we can move the php ini settings from Performance Tweaks to here or link to them?